### PR TITLE
bump version to 4.3.4-SNAPSHOT

### PR DESCRIPTION
**Pull request by albertshau**

### CODE REVIEW

**Comment by albertshau** [comment]
i'm not sure if the changes in cdap-distributions should have '-SNAPSHOT' in them or not.

**Review by anwar6953 — APPROVED** [review]

File: cdap-distributions/src/packer/cdap-sdk-ubuntu16-with-uri.json [hunk 1]
```diff
@@ -188,7 +188,7 @@
       "json": {
         "cdap": {
           "comment": "DO NOT PUT SNAPSHOT IN THE VERSION BELOW (UPDATE TO NEXT RELEASE), THIS CONTROLS CDAP COOKBOOK CODE",
-          "version": "4.3.3-1",
+          "version": "4.3.4-SNAPSHOT-1",
```
Comment (dereklwood) [inline cdap-distributions/src/packer/cdap-sdk-ubuntu16-with-uri.json]: leave this as `4.3.4-1`

File: cdap-distributions/src/packer/files/cdap-sdk.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "cdap": {
-    "version": "4.3.3-1"
+    "version": "4.3.4-SNAPSHOT-1"
```
Comment (dereklwood) [inline cdap-distributions/src/packer/files/cdap-sdk.json]: leave this as `4.3.4-1`

**Comment by dereklwood** [comment]
commented on the couple packer files that shouldn't have SNAPSHOT... otherwise, lgtm

**Pull request merged**